home work

olga-kri opened the pull request.

olga-kri committed:
Initial commit

OlgaLugacheva commented on src/Griffindor.java:
```diff
@@ -0,0 +1,54 @@
+    public void WhoIsTheBestOnGrifindor (Griffindor student){
+         //   if (student.getClass() == Griffindor.class) {
+         //       Griffindor griffindor = (Griffindor) student;
+          //  }
```
убираем закомментированный код

OlgaLugacheva commented on src/Griffindor.java:
```diff
@@ -0,0 +1,54 @@
+                ", храбрость " + bravery;
+    }
+
+    public void WhoIsTheBestOnGrifindor (Griffindor student){
```
названия методов с маленькой буквы

OlgaLugacheva commented on src/Griffindor.java:
```diff
@@ -0,0 +1,54 @@
+            this.nobleness = nobleness;
+            this.honor = honor;
+            this.bravery = bravery;
+            this.facultySkill = nobleness + honor + bravery;
```
ну я бы из поля это убрала и просто бы вынесла в метод - так будет правильнее, потому что это вычислимый показатель а не характеристика объекта

OlgaLugacheva replied:
относится и к остальным классам

OlgaLugacheva commented on src/Hogwarts.java:
```diff
@@ -0,0 +1,62 @@
+        this.magicalPowers = magicalPowers;
+        this.transgresson = transgresson;
+
+
```
лишние строчки стараемся не оставлять

OlgaLugacheva commented on src/Kogtevrun.java:
```diff
@@ -0,0 +1,73 @@
+                ", остроумие" + humor +
+                ", креативность" + creation;
+    }
+    public void WhoIsTheBestOnKogtevrun(Kogtevrun student){
```
переводим в нижний регистр

OlgaLugacheva commented on src/Slytherin.java:
```diff
@@ -0,0 +1,87 @@
+    public void WhoIsTheBestOnSlytherin(Slytherin student){
+        //   if (student.getClass() == Griffindor.class) {
+        //       Griffindor griffindor = (Griffindor) student;
+        //  }
```
см выше

olga-kri committed:
Замечания исправила -
1. Закоментированный код убрала: пыталась разобраться, что значит скастить класс, до конца не поняла.
2. Названия методов с маленькой буквы - перевела в нижний регистр.
3. Убрала из поля классов сумму личных качств - перенесла в метод.
4. Пустые строчки почистила.
